test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -71,14 +70,19 @@ def _get_safety_ignore_list(image_uri):
 
 
 @pytest.mark.canary("Run safety tests regularly on production images")
+@pytest.mark.skipif(not is_dlc_cicd_context(), reason="Skipping test because it is not running in dlc cicd infra")
```

Comment:
I couldn't develop a clean method to do so. Perhaps this can be addressed in another PR that takes care of a 'local' context for build. For now, we can retain to running the tests in cicd infra.